src/MentionsInputInputProps.ts

```
@@ -0,0 +1,157 @@
+  isSuggestionsOpened,
+  suggestionsOverlayId,
+  inlineAutocompleteLiveRegionId,
+  focusIndex,
+  onScroll,
+  onChange,
+  onSelect,
+  onKeyDown,
+  onBlur,
+  onCompositionStart,
+  onCompositionEnd,
+}: BuildMentionsInputInputPropsArgs<Extra>): InputComponentProps => {
```

ESLint

[error] Refactor this function to reduce its Cognitive Complexity from 17 to the 15 allowed.

(sonarjs/cognitive-complexity)